libs/signal-store/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "@mini-rx/signal-store",
+    "version": "0.0.20",
+    "author": "Florian Spier <mail@florian-spier.be> (https://florian-spier.be)",
+    "license": "MIT",
+    "homepage": "https://github.com/spierala/mini-rx-store",
+    "readme": "https://github.com/spierala/mini-rx-store/blob/master/README.md",
+    "description": "Signal Store for Angular",
+    "keywords": [
+        "Angular",
+        "Signal",
+        "RxJS",
+        "Observable",
+        "Redux",
+        "Store",
+        "State Management"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/spierala/mini-rx-store.git"
+    },
+    "bugs": {
+        "url": "https://github.com/spierala/mini-rx-store/issues"
+    },
+    "peerDependencies": {
+        "@angular/common": "^16.0.0",
+        "@angular/core": "^16.0.0",
```

Comment:
As a lib dev, it might be more convenient, to go for a wider range of Angular in your `peerDependencies`.

f.e.: >=16 , when the api stays stable in Angular, you are not (friendly) obliged to release a new version.

Reply:
That is a good point!

I relaxed the Angular range.

```
 "peerDependencies": {
        "@angular/core": ">= 16.0.0",
        "rxjs": "^7.0.0"
    },
    "dependencies": {
        "@mini-rx/common": "^0.0.6",
        "tslib": "^2.3.0"
    },
```

I also made `@mini-rx/common` a dependency (not peer dep anymore).

It could help in some IDEs to not suggest imports from "@mini-rx/common".

Ideally the common code would be compiled into the signal store lib, but that does not seem to be possible: https://stackoverflow.com/questions/77243766/angular-library-make-local-typescript-dependency-part-of-the-compilation/77268952#77268952